### DISCUSSION TITLE
Ask for feature: script the ui of the terminal

### DISCUSSION THREAD

**Discussion by throwoutofcoffeeexception** [microsoft/terminal#13619] [Q&A] [answered]
Hello Team,
is it somehow possible to script the ui of the terminal. The Use case is that I use it for my developer work everyday. I use several tabs to start the different servers.... Can I script this somehow. So i can start the setup with one mouseclick....?
best regards

**Comment by zadjii-msft** [comment, marked as answer]
You mean something like [`multipleActions`](https://docs.microsoft.com/en-us/windows/terminal/customize-settings/actions#run-multiple-actions)?
